lib/cli.js

```diff
@@ -182,6 +183,15 @@ const cli = {
 
             log.info(JSON.stringify(fileConfig, null, "  "));
             return 0;
+        } else if (currentOptions.info) {
+            log.info("\n Environment Information:-");
+            envinfo
+                .run({
+                    System: ["OS", "CPU"],
+                    Binaries: ["Node", "Yarn", "npm"],
+                    npmGlobalPackages: ["eslint"]
+                })
+                .then(log.info);
```

Comment:
Does `envinfo` have a synchronous API option? It looks like we'd have a dangling promise when it's called [here](https://github.com/eslint/eslint/blob/master/bin/eslint.js#L81).